src/TextForEvent.js

```diff
@@ -101,6 +110,12 @@ function textForCallInviteEvent(event) {
     return senderName + " placed a " + type + " call." + supported;
 };
 
+function textForThreePidInviteEvent(event) {
+    var senderName = event.sender ? event.sender.name : event.getSender();
+    return senderName + " sent an invitation to " + event.getContent().display_name +
+     " to join the room.";
```

Comment:
Okay - this is more one for the designers anyway. From a code PoV this looks fine to me.